api/create_distributed_hypertable.md

```diff
@@ -95,4 +95,16 @@ be removed in a future release. To avoid such inconsistency, we do not yet
 recommend using `replication_factor` > 1, and instead rely on physical
 replication of each data node if such fault-tolerance is required.
 
+## Troubleshooting
```

Comment:
Generally speaking, I prefer to have Troubleshooting as its own section, rather than tacked on the bottom of other content. Is that possible here? See timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md for an example.